cmd/werf/common/common.go

```diff
@@ -1073,6 +1064,45 @@ func GetNamespace(cmdData *CmdData) string {
 	return *cmdData.Namespace
 }
 
+// FIXME: inconsistent behaviour:
+// "WERF_SET_TEST=test=1val" and "--set "test=2val"" results in "test: 2val", but
+// "WERF_SET_TEST=test=3val" and "--set "test=2val"" results in "test: 3val";
+// "...test=true" and "...test=false" in any order results in "test: true"
+func GetSet(cmdData *CmdData) []string {
+	envSet := predefinedValuesByEnvNamePrefix("WERF_SET_", "WERF_SET_STRING_", "WERF_SET_FILE_")
+	result := util.UniqStrings(append(envSet, *cmdData.Set...))
+	sort.Strings(result)
+	return result
+}
+
+func GetSetString(cmdData *CmdData) []string {
+	envSetString := predefinedValuesByEnvNamePrefix("WERF_SET_STRING_")
+	result := util.UniqStrings(append(envSetString, *cmdData.SetString...))
+	sort.Strings(result)
```

Comment:
Sort in the predefinedValuesByEnvNamePrefix function by the environment variable names.

Reply:
sort removed from all common.GetSet*/Get*Values functions